ui/components/app/confirm/info/row/divider.tsx

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 
 export const ConfirmInfoRowDivider: React.FC = () => {
   return (
-    <div
-      style={{
-        height: '1px',
-        backgroundColor: 'var(--color-border-muted)',
-        // Ignore the padding from the section.
-        marginLeft: '-8px',
-        marginRight: '-8px',
-      }}
-    ></div>
+    <div style={{ paddingTop: '8px', paddingBottom: '8px' }}>
+      <div
+        style={{
+          height: '1px',
+          backgroundColor: 'var(--color-border-muted)',
+          // Ignore the padding from the section.
+          marginLeft: '-8px',
+          marginRight: '-8px',
+        }}
+      />
+    </div>
   );
```

Comment:
Hey @vinistevam : any reason to prefer to wrap div in another ? Is it possible that we apply padding to one div only.

Reply:
I tried it at first, but the inner div is the divider itself, so if I apply padding, it increases the divider. Additionally, it loses the margin left and right.
![image](https://github.com/user-attachments/assets/33fd73e3-2cee-40d7-8411-62a31ab11723)